### DISCUSSION TITLE
Are we getting Isar v4 or not?

### DISCUSSION THREAD

**Discussion by theprantadutta** [isar/isar#1590] [Help]
Thank you for creating this really awesome library, but I haven't seen a new release of this library since last August.

So, I was wondering should I use this project going forward? right now, I have to use a lot of outdated libraries already to use this library.
If we are not going to get a new version, then I think I should look for an alternative, that said, I really like this project, and I want it to move forward, but I can't keep using outdated libraries to use this in future.

Once Again, Thanks a lot for creating this really awesome library.

**Comment by vicenterusso** [comment]
You should try the [community fork](https://github.com/isar-community/isar), with updated dependencies

**Comment by bgintzairspace** [comment]
@vicenterusso Generally speaking is the plan to still fork the community version back into Simon's version or is the plan to keep it separate?

**Reply by vicenterusso** [reply]
The plan is to keep Isar alive. I'd love to see Simon back and keep working on`isar/isar`, but I doubt he will take any code from the community fork
